Cookbook/Technical-Documents/ReleaseProcess.md

```diff
@@ -23,20 +23,24 @@ Release duties have priority over regular squad duties. Please inform your squad
 
 There are usually two release engineers working at any given time. It goes without saying that both engineers need to work together and that constant feedback is vital.
 
-### 2. Release step-by-step
+## 2. Release step-by-step
 
 **Phase 1: Initiation**
 <br/>	*It starts at the end of the sprint (typically when the new sprint starts on Monday)*
 
-1. Cut a release branch:  Create a new branch from develop and push to origin (e.g `release/3.2.0`).
+1. Cut a release branch by creating a new branch from develop and push it to `origin`
+  * For releases concerning both Babylon and Telus at once (most common case), name it `release/{version}` (e.g. `release/3.2.0`)
+  * For releases specific to only Babylon or only Telus, or for releases for other apps (Bupa, NHS111, ...), name it `release/{appname}/{version}` (e.g. `release/bupa/3.2.0`)
 1. Create a slack channel to discuss anything relative to the release (e.g. `ios_release_3_2_0`).
 1. Bump the release version by triggering its command (eg. `/release babylon:3.2.0`) in `#ios-build` (you can run the command every time you want to upload a new build).
-    * This creates a test Tesflight build (try to make one as early as possible so that you can catch issues like missing/expired certificates or profiles and any other production build errors early).
+  * This creates a test TestFlight build (try to make one as early as possible so that you can catch issues like missing/expired certificates or profiles and any other production build errors early).
```

Comment:
```suggestion
  * This creates a TestFlight build (try to make one as early as possible so that you can catch issues like missing/expired certificates or profiles and any other production build errors early).
```